numbat-cli/src/main.rs

```diff
@@ -24,12 +24,41 @@ use rustyline::{
     Validator,
 };
 use rustyline::{EventHandler, Highlighter, KeyCode, KeyEvent, Modifiers};
+use serde::Deserialize;
+use toml;
 
 use std::io::IsTerminal;
 use std::path::PathBuf;
 use std::sync::{Arc, Mutex};
 use std::{fs, thread};
 
+#[derive(Deserialize, PartialEq, Eq)]
+#[serde(rename_all = "kebab-case")]
+enum ExchangeRateFetchingPolicy {
+    Prefetch,
+    FetchOnFirstUse,
+    DontFetch,
```

Comment:
Need to think about best names here (and also for the names of config options themselves).

Reply:
I like what you have.

Reply:
Thanks, here are my concerns though, if you want to mull on them some more:
- `"dont-fetch"` looks kind of weird to be honest
- `exchange-rate-fetching-policy` is kind of long
- maybe `show-intro` would be clearer than `quiet`, but we're already using `quiet` as a command-line option so I'm leaning towards just using `quiet`